marlowe-playground-client/src/Types.purs

```diff
@@ -216,6 +218,12 @@ _haskellEditorKeybindings = _Newtype <<< prop (SProxy :: SProxy "haskellEditorKe
 _marloweEditorKeybindings :: Lens' FrontendState KeyBindings
 _marloweEditorKeybindings = _Newtype <<< prop (SProxy :: SProxy "marloweEditorKeybindings")
 
+_activeMarloweDemo :: Lens' FrontendState String
+_activeMarloweDemo = _Newtype <<< prop (SProxy :: SProxy "activeMarloweDemo")
```

Comment:
This is a small, hopeless prayer that one day PureScript gets type applications.